Automattic/jetpack
# Experiment: try search results in Gutenberg

We currently deliver Jetpack Search results through an overlay. In this PR, we'll explore whether we could offer a Gutenberg search results block.

#### Changes proposed in this Pull Request:

* Add a new search results block, and connect it up to Jetpack Search via the API.

#### Jetpack product discussion



There are no plans to merge this work to Jetpack at this stage.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






1. Create a new page on your test site.
2. Add the 'Search Results (beta)` and `Search Filters (beta)` blocks to it.
3. Start a search using `?search=` in the query string. For example, `?search=art` on the default 2020 theme site.
4. Check out the two rendered blocks. The 'filters' block should change in step with the results block.

#### Proposed changelog entry for your changes:

There are no plans to merge this work to Jetpack at this stage.
